Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_v2.py

```diff
@@ -0,0 +1,1791 @@
+from datetime import date
+from typing import Any, Dict, Tuple, Optional
+
+import json
+import traceback
+import urllib3
+
+from CommonServerPython import *  # pylint: disable=unused-wildcard-import
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+CY_GENERAL_DATE_FORMAT = '%Y-%m-%d %H:%M:%S'
+CY_UNIQUE_DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
+MAX_INCIDENTS_TO_FETCH = 35
+MAX_EVENTS_TO_DISPLAY = 20
+
+SEVERITIES = ['Low', 'Medium', 'High', 'Critical']
+ACCESSED_STATUS = ['penetrated', 'accessed', 'executed completely', 'exfiltrated', 'completed']
+ENDPOINT_DICT = {'web-gateway': 'browsing',
+                 'exfiltration': 'dlp',
+                 'email-gateway': 'mail',
+                 'endpoint-security': 'edr',
+                 'waf': 'waf',
+                 'kill-chain': 'apt',
+                 'immediate-threats': 'immediate-threats',
+                 'phishing-awareness': 'phishing',
+                 'lateral-movement': 'hopper'
+                 }
+
+
+class Client(BaseClient):
+    """
+    Client for Cymulate RESTful API.
+
+    Args:
+          base_url (str): Cymulate server url.
+          token (str): Cymulate access token.
+          proxy (bool): specifies if to use XSOAR proxy settings.
+    """
+
+    def __init__(self, base_url: str, token: str, proxy: bool, **kwargs):
+        super().__init__(base_url=base_url, proxy=proxy, **kwargs)
+        self.headers = {'x-token': token,
+                        'accept': '*/*',
+                        'Content-Type': 'application/json'
+                        }
+
+    def validate(self):
+        """ Helper function for test_module that creates a simple API call"""
+        return self._http_request(method='GET',
+                                  url_suffix='/user/modules',
+                                  headers=self.headers,
+                                  resp_type='response')
+
+    def start_assessment(self, endpoint: Optional[str], data: dict):
+        """ Start new assessment.
+
+        Args:
+            endpoint (str): Cymulate's endpoint to start the assessment on.
+            data (dict): Dictionary containing all relevant data for running new assessment.
+        """
+        return self._http_request(method='POST',
+                                  url_suffix=f'{endpoint}/start',
+                                  headers=self.headers,
+                                  data=build_data(data))
```

Comment:
for future reference, instead of converting the `data` object to a string, you can pass it to `requests` and let it handle the conversion.
both `data` and `json` parameters support a dictionary input:
![image](https://user-images.githubusercontent.com/30797606/110260683-e56f0b00-7fb5-11eb-93ae-9375f49bfef7.png)